pennylane/drawer/draw.py

```diff
@@ -306,8 +306,9 @@ def draw_mpl(
             Default ``None`` will omit parameters from operation labels.
         style (str): visual style of plot. Valid strings are ``{'black_white', 'black_white_dark', 'sketch',
             'sketch_dark', 'solarized_light', 'solarized_dark', 'default'}``. If no style is specified, the
-            ``'black_white'`` style will be used. Setting style does not modify matplotlib global plotting settings.
-            If ``None``, the current matplotlib settings will be used.
+            the current matplotlib settings will be used, and the initial default is 'black_white'. If you would

```

Comment:
I'm not sure about the phrasing "current matplotlib settings".  Because for me, I would interpret "current matplotlib settings" as `rcParams`.

Maybe something along the lines of 

```
the global style set with :func:`~.use_style` will be used.
```